libs/coin-polkadot/tsconfig.json

```diff
@@ -4,8 +4,6 @@
     "declarationMap": true,
     "sourceMap": true,
     "strict": true,
-    "noImplicitAny": false,
```

Comment:
Very tempting to apply the no-explicit-any rule at the same time. :slightly_smiling_face:
Did you test with that rule? Just curious of the number of errors to fix for Polkadot, if small maybe it could be done too, otherwise can be done separately of course.

Reply:
Unfortunately, in the `bisontrails` I was not able to avoid declaring a lot of them :(

Reply:
Aah the API indeed...  Most of the backend APIs we use are typed but not all of them :pensive: 
No problem anyway, thanks.

Reply:
Could be worth try to apply the `no-explicit-any` rule after applying my comments below